### PR TITLE
Create rany2.yaml

### PR DESCRIPTION
**Pull request by rany2**

### CODE REVIEW

**Comment by rany2** [comment]
image

**Comment by 6r1d** [comment]
Hi. You can leave a comment here so we know you've changed your mind, though.
Not sure it's ready, but it's something.
https://codeberg.org/rms-support-letter/not-rms-support-letter/issues/1
Thanks.

**Comment by Aspie96** [comment]
@rany2 could you try to email them?

**Comment by rany2** [comment]
@Aspie96 https://codeberg.org/rms-support-letter/not-rms-support-letter/issues/1#issuecomment-189719

I did. Hopefully that email is monitored, I really regret signing the open letter and feel like I've been misled

**Comment by Aspie96** [comment]
@rany2 

Thank you for sharing your experience BTW

Would you please consider writing a more complete comment about what mislead you and why you changed your mind (unless you signed the wrong letter by mistake)?
I think a discussion about this could help a lot!

(This is true in general. The prospective of those who change their views is often very valuable).

**Comment by rany2** [comment]
@Aspie96 

I attempted to do some research before signing the open letter, so I searched online for "richard stallman misogyny" (https://duckduckgo.com/?q=richard+stallman+misogony) and a couple of other searches regarding the claims by the open letter. All the searches appeared to return trustworthy news articles that were anti-Stallman (and in fact it still does). I didn't know that the claims against Stallman in the open letter and the news articles were a bunch of lies and so I signed the letter because I legitimately thought he was as dangerous as they were portraying him to be. 

And just to be clear I find the way Stallman got reinstated to be pretty shady and not transparent of the FSF. If the open letter cited some non-BS reason like "transparency" I wouldn't have minded it and probably wouldn't have signed this support letter.  

tl;dr the open letter straight up lied and distorted the truth to get me to sign their letter and that is something I very much didn't appreciate of them

**Comment by Aspie96** [comment]
Thank you!

And this is the reason those signing the Open Letter should be approached with evidence and we should be kind to them.

Because many of them, just like you, have been mislead. But have an open mind.
